chia/util/streamable.py

```diff
@@ -269,8 +269,19 @@ def function_to_parse_one_item(cls: Type[cls.__name__], f_type: Type):  # type:
 
     @classmethod
     def parse(cls: Type[cls.__name__], f: BinaryIO) -> cls.__name__:  # type: ignore
-        values = [parse_f(f) for parse_f in PARSE_FUNCTIONS_FOR_STREAMABLE_CLASS[cls]]
-        return cls(*values)
+        # Create the object without calling __init__() to avoid unnecessary post-init checks in strictdataclass
+        obj = object.__new__(cls)
```

Comment:
Can you use type annotations for all of these variables? The code is hard to read otherwise

Reply:
And they don't have a performance impact

Reply:
Sure, will add them.